### DISCUSSION TITLE
Is it possible to completely disable anything Slack related within outline?

### DISCUSSION THREAD

**Discussion by ovizii** [outline/outline#7007] [General]
I mean completely remove and possibility of connecting anything to slack?

![image](https://github.com/outline/outline/assets/837902/90064d86-bb4b-4194-b5f8-5cf1620d3d3e)

**Comment by tommoor** [comment]
Not currently, do you have no `SLACK_` config in your environment? If not then I'd probably consider this a bug.

**Reply by ovizii** [reply]
When I first set up my outline instance, I used SLACK to set up my first user. I then moved to sign-in via email link and now finally to OIDC. I have removed all SLACK_ related configs from .env and from my docker-compose.yml file. 

The “old” user which was using Slack as authentication has been suspended. The current OIDC user has a different email address. I have just deleted that old SLACK user, but it did not change anything.

This is not a dealbreaker, I was just checking if I was doing something wrong.
